src/support/logger/logger.h

```diff
@@ -51,6 +51,7 @@ Logger* logger_getDefault();
 
 // Thread safe. `logger` may be NULL, in which case glib's logging
 // functionality will be used.
+__attribute__((__format__ (__printf__, 6, 7)))
```

Comment:
A comment about what this does would be helpful.

Reply:
Done